soc/arm/nxp_lpc/lpc55xxx/soc.c

```diff
@@ -77,6 +77,11 @@ static ALWAYS_INLINE void clock_init(void)
 	RESET_PeripheralReset(kHSLSPI_RST_SHIFT_RSTn);
 #endif
 
+#if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(wwdt0), nxp_lpc_wwdt, okay)
```

Comment:
why us DT macro here v CONFIG_WDT_MCUX_WWDT?

Reply:
I2C and SPI clocks are both set up using DT macros, so I did the same thing here. I'm not sure if DT macros are better than CONFIG_WDT_MCUX_WWDT option, should I change it?